internal/xds/translator/extensionserver_test.go

```diff
@@ -202,6 +202,17 @@ func (t *testingExtensionServer) PostHTTPListenerModify(_ context.Context, req *
 
 // PostTranslateModifyHook inserts and overrides some clusters/secrets
 func (t *testingExtensionServer) PostTranslateModify(_ context.Context, req *pb.PostTranslateModifyRequest) (*pb.PostTranslateModifyResponse, error) {
+	for _, cluster := range req.Clusters {
+		if edsConfig := cluster.GetEdsClusterConfig(); edsConfig != nil {
+			if strings.Contains(edsConfig.ServiceName, "fail-close-error") {
```

Comment:
would be good to have some comments in here explaining what this is simulating

Reply:
Done.